src/renderer/components/app.tsx

```diff
@@ -112,9 +116,14 @@ export class App extends React.Component {
         preload: true,
       })
     ]);
+
+    setTimeout(() => {
```

Comment:
how about if `AllowedResources` had some `loading` observable and we would display loader while it's true?

Reply:
How about it is loading only during the first load?